transcribe.py

```diff
@@ -158,6 +212,7 @@ def transcribe_all():
 		logger.info("Starting Transcription of Video File: %s" % video_file)
 		comment = transcribe_video_file(video_file)
 		output[video_file] = comment
+		convert_transcribe_to_srt(video_file)
```

Comment:
It would be nice to do a best-effort post-processing of comments within a try-catch block, so that if the post-processing of a single transcript fails  we can still move on to the rest.